index.html

```diff
@@ -0,0 +1,464 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+    <script src="index.js" defer></script>
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <a name="top"></a>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" title="menu" class="menu-bars" id="menuBars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
+
+      <nav id="menuMobile" class="menu-mobile">
+        <ul class="list-menu">
+          <li class="list-menu-item"><a href="#top">Hello</a></li>
+          <li class="list-menu-item"><a href="#projects">Portfolio</a></li>
+          <li class="list-menu-item"><a href="#about-me">About</a></li>
+          <li class="list-menu-item"><a href="#contact">Contact</a></li>
+        </ul>
+        <i
+          id="close-button"
+          class="fas fa-times fa-2x cross invisible"
+          onclick="toggleMobileMenu()"
+        ></i>
+      </nav>
```

Comment:
You have done a fantastic job on upgrading your previous design as the @VICTOR-CHUKWU requested! I would suggest one change to fully mimic the Figma design.

As you can see in the image below, there are some white separations between the `<li>` elements. Please add them to your design to set your project on fire 🔥🔥🔥

![Captura](https://user-images.githubusercontent.com/15807343/147364141-a4d980f3-f25d-457a-af5e-0875b61accfc.PNG)